db/models/product.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const {STRING, TEXT, DOUBLE, INTEGER} = require('sequelize')
+
+module.exports = db => db.define('products', {
+  title: STRING,
+  description: TEXT,
+  price: DOUBLE,
+  invQty: INTEGER,
+  image: STRING
+})
+
+module.exports.associations = (Product, {User, Order, Review, Category}) => {
+  Product.belongsToMany(User, {through: Order})
```

Comment:
I'm not sure about this first association. What is it intended to do? 

You are probably going to need another association here, so that an order has many products, and the order can have Sequelize methods that allow products to be added to it.